Sources/XcodeProj/Objects/Targets/PBXTarget.swift

```diff
@@ -190,14 +210,21 @@ public class PBXTarget: PBXContainerItem {
         }
 
         dictionary["dependencies"] = .array(dependencyReferences.map { .string(CommentedString($0.value, comment: PBXTargetDependency.isa)) })
+        if let fileSystemSynchronizedGroupsReferences {
+            dictionary["fileSystemSynchronizedGroups"] = .array(fileSystemSynchronizedGroupsReferences.map { fileSystemSynchronizedGroupReference in
+                let fileSystemSynchronizedGroup: PBXFileSystemSynchronizedRootGroup? = fileSystemSynchronizedGroupReference.getObject()
+                return .string(CommentedString(fileSystemSynchronizedGroupReference.value, comment: fileSystemSynchronizedGroup?.name))
```

Comment:
When performing a read / write test of an Xcode 16 project that uses this it ends up omitting the comment string, possibly we're missing reading it?

e.g. 

```diff
                        fileSystemSynchronizedGroups = (
-                               FB50EE602C5A922900C43B53 /* FolderRefs */,
+                               FB50EE602C5A922900C43B53,
                        );
```

Reply:
I was accessing the wrong property.